webpack.build.config.js

```diff
@@ -12,8 +11,16 @@ if (!isProd) {
   /*
   Till we do not import css from react components, we need to have
   this workaround for Hot reload to work with global css.
+  We are not using "glob" to get all files in "css" folder as order of files matters.
   */
-  const globalCssPathnames = glob.sync('./app/public/css/**/*.css') || []
+  const globalCssPathnames = [
```

Comment:
I find this name a bit misleading as the paths are obviously relative. I know this is a preexisting name but I don't really see why we couldn't just name it cssPaths or something like that.

Also, I'd consider retaining the original fetching of all css files and asserting that the lists are equal (in terms of elements contained). Otherwise I can see future ourselves scratching our heads why some css file is not being picked up, and such assertion seems like a good fool-proof measure to prevent any such surprises